grammar/grammar.y

```diff
@@ -597,107 +582,107 @@ for_expression
 primary_expression
     : '(' primary_expression ')'
       {
-        
+        $$ = $2
       }
     | _FILESIZE_
       {
-        
+        $$ = data.Expression{Left: data.Keyword{Name: "filesize"}}        
       }
     | _ENTRYPOINT_
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _INTEGER_FUNCTION_ '(' primary_expression ')'
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _NUMBER_
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _DOUBLE_
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _TEXT_STRING_
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _STRING_COUNT_
       {
-        
+        $$ = data.Expression{Unsupported: true}
       }
     | _STRING_OFFSET_ '[' primary_expression ']'
```

Comment:
Likewise here and for `_STRING_COUNT_`, regarding referring back to the `data.String`